examples/models/toolbars.py

```diff
@@ -1,3 +1,12 @@
+'''This example shows multiple ways to place the toolbar and the x and y axes with respect to the plot
```

Comment:
minor edit and line length
```suggestion
''' This example shows multiple ways to place the toolbar and the x and y axes
with respect to the plot.
```